Isodose/Logic/vtkMRMLIsodoseNode.h

```diff
@@ -141,6 +146,9 @@ class VTK_SLICER_ISODOSE_LOGIC_EXPORT vtkMRMLIsodoseNode : public vtkMRMLNode
   /// Whether use relative isolevels representation
   /// for absolute dose (Gy) and unknown units or not
   bool RelativeRepresentationFlag;
+
+  /// Use segmentation flag for isodose models
+  bool SegmentationFlag;

```

Comment:
Quite unfortunate name I think, because only vaguely refers to its purpose. How about `UseSegmentationToStoreIsodoseSurfaces` ?

Reply:
I will rename it.